internal/store/repo/aurora_health_check_repo.go

```diff
@@ -0,0 +1,58 @@
+package repo
+
+import (
+	"context"
+	"fmt"
+	"github.com/kong/pg-aurora-client/internal/store/ent"
+	"time"
+)
+
+type AuroraHealthCheckRepo struct {
+	client *ent.Client
+}
+
+func NewAuroraHealthCheckRepo(client *ent.Client) *AuroraHealthCheckRepo {
+	return &AuroraHealthCheckRepo{client: client}
+}
+
+func (repo AuroraHealthCheckRepo) Upsert(ctx context.Context, id *int) (*ent.AuroraHealthCheck, error) {
```

Comment:
The pointer to an integer seems strange... this would also panic if it's nil. Why's it need to be a pointer in the first place?

This comment applies to the other methods as well.